.github/workflows/binskim.yaml

```diff
@@ -0,0 +1,128 @@
+name: Release Binary Hardening checks
```

Comment:
should we add a codeql file as well for the tarfs module, similar to https://github.com/kata-containers/kata-containers/pull/10930/files ?